stdlib/asm/collections/mmr.masm

```diff
@@ -107,6 +108,33 @@ export.get
   end
 end
 
+#! Given the num_leaves of a Mmr returns the num_peaks.
```

Comment:
minor nit: I'd use `MMR` instead of `Mmr`.

Reply:
renamed